Add information about Bluetooth interference (#23521)
home-assistant/home-assistant.github.io: source/_integrations/bluetooth.markdown

While newer integrations can share the Bluetooth Adapter, some legacy integrations require exclusive use of the adapter. Enabling this integration may prevent an integration that has not been updated to use newer methods from functioning.

Deleting the config entry for this integration will release control of the adapter and allow another integration to gain exclusive use of the Bluetooth adapter. If you have manually added `bluetooth:` to your `configuration.yaml`, you must also remove it to prevent the configuration from being recreated.

## Bluetooth interference with other devices

> Devices that are using the 2.4 GHz band, like Wifi, Zigbee and USB3 devices (and its cable connections) are known to affect Bluetooth reception. Especially external SSD-drives with USB3 cables are known to block the Bluetooth signal. Also, metal casings can decrease Bluetooth performance of internal Bluetooth Adapters. The following tips may improve reception of the Bluetooth Adapter.

Reviewer: ```suggestion
Devices that are using the 2.4 GHz band, like Wi-Fi, Zigbee, and USB3 devices (and their cable connections) are known to affect Bluetooth reception. Especially external SSD drives with USB3 cables are known to block the Bluetooth signal. Also, metal casings can decrease the Bluetooth performance of internal Bluetooth Adapters.

The following tips may improve reception of the Bluetooth Adapter:
```

> - Try to place USB3 devices (SSD, etc.) as far away as possible from your Bluetooth Adapter, e.g., by using an extension cable.
> - Use an USB3 extension cable with proper shielding and ferrite clamps
> - Use a (good quality) external Bluetooth adapter with antenna

Reviewer: ```suggestion
- Try to place USB3 devices (SSD, etc.) as far away as possible from your Bluetooth Adapter, e.g., by using an extension cable.
- Use a USB3 extension cable with proper shielding and ferrite clamps.
- Use a (good quality) external Bluetooth adapter with an antenna.
```
